week3/src/components/SelectLevel.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from  "react";
+import styled from 'styled-components';
+
+const SelectLevel = () => {
+    let level = ['Easy', 'Normal', 'Hard'];
+
+    let [btnActive, setBtnActive] = useState("");
+
+    const toggleActive = (e) => {
+        setBtnActive(() => {
+        return e.target.value;
+        });
+    };
```

Comment:
이 부분에선 함수형 업데이트를 할 필요는 없어보여요!! 불필요한 함수형 업데이트는 가독성을 저해합니다!!

```suggestion
    const toggleActive = (e) => {
        setBtnActive(e.target.value);
    };
```